src/main/java/net/snowflake/ingest/utils/Constants.java

```diff
@@ -49,12 +49,14 @@ public class Constants {
   public static final int BLOB_FILE_SIZE_SIZE_IN_BYTES = 8;
   public static final int BLOB_CHECKSUM_SIZE_IN_BYTES = 8;
   public static final int BLOB_CHUNK_METADATA_LENGTH_SIZE_IN_BYTES = 4;
-  public static final long THREAD_SHUTDOWN_TIMEOUT_IN_SEC = 300L;
+  public static final long THREAD_SHUTDOWN_TIMEOUT_IN_SEC = 30L;
```

Comment:
Why is this change needed?

Reply:
Yes accidentally made its way to PR, had removed it locally already, thanks!